### ISSUE TITLE
Failed to compile nuxt-prismic-slices project

### ISSUE THREAD

**Issue by pablogiralt** [prismicio/slice-machine#21]
Hi, I have gone through the slicemachine tutorial at https://www.slicemachine.dev/documentation/installing-npm

Once project is set up I run yarn dev and I get this compilation error:

```
ERROR  Failed to compile with 1 errors
This dependency was not found:
To install it, you can run: npm install --save @prismicio/vue/components/common
```

**Comment by lihbr** [comment]
Hey @pablogiralt, thanks for your input!

This is a known bug (due to us renaming our Vue SDK package name) that is awaiting for a merge to be fixed on #15 (btw cc @hypervillain)

In the meantime you can uninstall `prismic-vue` and install `@prismicio/vue`:
```bash
$ npm uninstall prismic-vue && npm install @prismicio/vue
```

Let us know how it goes~!

**Comment by hypervillain** [comment]
This should work, closing for now
